guide/transfer-to-your-own-robot.md

```diff
@@ -80,11 +80,11 @@ system for the e-puck and the Hemisson robot. Samples are located in the
 
 #### Developing a custom library
 
-Unlike the remote control system, the cross-compilation system requires that the
-source code of your Webots controller be recompiled using the cross-compilation
-tools specific to your own robot. You will also need to rewrite the Webots
-include files to be specific to your own robot. In simple cases, you can just
-rewrite the Webots include files you need, as in the "hemisson" example. In more
+Unlike the remote control system, the cross-compilation system requires the
+source code of your Webots controller to be recompiled using the cross-compilation
+tools specific to your own robot. You will also need to rewrite the Webots include-files
+to be specific to your own robot. In simple cases, you can just
+rewrite the Webots include-files you need, as in the "hemisson" example. In more
```

Comment:
I would put a dash between include and files, "include files" is a very common CS terminology.